flink-ml-core/src/main/java/org/apache/flink/ml/common/broadcast/BroadcastUtils.java

```diff
@@ -57,8 +56,9 @@ public class BroadcastUtils {
      * @param bcStreams map of the broadcast data streams, where the key is the name and the value
      *     is the corresponding data stream.
      * @param userDefinedFunction the user defined logic in which users can access the broadcast
-     *     data streams and produce the output data stream. Note that users can add only one
-     *     operator in this function, otherwise it raises an exception.
+     *     data streams and produce the output data stream. Note that though users can add more than
+     *     one operator in this logic, but only the operator that generates the result stream can
+     *     contain a rich function and access the broadcast variables.
```

Comment:
Would it be useful to include the following:

Other operators will encounter NPE when accessing the broadcast variables.